### PR TITLE
Styling/media queries

### PR DESCRIPTION
**Pull request by attom2**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the new behavior?

- Adds margin to header for desktop view,
- Makes nav buttons smaller for small phones
- Removes margin on best streak and winning streak, makes font smaller on smaller phones

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
image
image

### CODE REVIEW

File: src/Components/Header/Header.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 header {
-  background: linear-gradient(to top left, #FF0000 4%, #CC0000 100%);
+  background: #FF0000;
```
Comment (MarchCorbin) [inline src/Components/Header/Header.scss:2]: Whats the logic for making the colors more flat?

File: src/Components/Header/Header.scss [hunk 2]
```diff
@@ -69,3 +69,19 @@ button {
...
+
+@media screen and (min-width: 750px){
+  header {
+    width: 50%;
+    margin-left: 25%;
+    background: #FF0000;
+  }
+}
+
+@media screen and (max-height: 600px) {
```
Comment (MarchCorbin) [inline src/Components/Header/Header.scss:81]: Good idea accommodating for really small screens!

File: src/Components/Header/Header.scss (hunk 1 above)
Comment (attom2) [reply src/Components/Header/Header.scss:2]: So the header style matches the pokedex style

**Comment by emontealvo** [comment]
I like the thoughtful use of media queries.

**Pull request merged**